gateway/src/main/webapp/app/entities/MSUserService/ms-members/ms-member-update.component.ts

```diff
@@ -40,15 +40,15 @@ function parentSalesforceIdConditionallyRequiredValidator(): ValidatorFn {
 function clientIdValidator(): ValidatorFn {
   return (control: AbstractControl): { [key: string]: boolean } | null => {
     if (control.value !== undefined && isNaN(control.value)) {
-      const clientIdValue = control.value;
+      var clientIdValue = control.value;
       if (clientIdValue.startsWith('APP-') && clientIdValue.match(/APP-[A-Z0-9]{16}$/)) {
         return null;
-      } else if (clientIdValue.match(/[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}$/)) {
+      } else if (clientIdValue.match(/[A-Z0-9]{4}-[A-Z0-9]{4}\-[A-Z0-9]{4}-[A-Z0-9]{4}$/)) {
```

Comment:
Hey @Camelia-Orcid is the `\` ok here? if so, why not in the others?

Reply:
Fixed. It was a typo.